R/algorithm.R

```diff
@@ -108,7 +108,7 @@ algorithm <- function() {
     no_potential_pcos = list(
       register = NA,
       title = "No potential PCOS",
-      logic = "koen == 2 AND atc =~ '^A10BA02$' AND ((date - foed_dato) < 40 OR indication_code %in% c('0000092', '0000276', '0000781'))",
+      logic = "koen == 2 AND atc =~ '^A10BA02$' AND ((date - foed_dato) / 365.25 < 40 OR indication_code %in% c('0000092', '0000276', '0000781'))",
```

Comment:
Aaahh, I see the issue here. Hmmmmm. We need to define what `365.25` and `40` are. A date minus a date gives a duration or interval. And an interval can't just magically do math on numbers.